NuGet.Services.Dashboard.Operations/Tasks/DashBoardTasks/AzureManagementTasks/CreateTrafficManagerStatusOverviewReportTask.cs

```diff
@@ -29,36 +29,48 @@ public override void ExecuteCommand()
             request.Headers.Add("x-ms-version: 2014-02-01");
             request.PreAuthenticate = true;
             request.Method = "GET";
-            WebResponse respose = request.GetResponse();
+            WebResponse response = request.GetResponse();
             //Schema of the response would be as specified in http://msdn.microsoft.com/en-us/library/azure/hh758251.aspx
-            Console.WriteLine(respose);
-            using (var reader = new StreamReader(respose.GetResponseStream()))
-            {               
+            using (var reader = new StreamReader(response.GetResponseStream()))
+            {
+                var responseContent = reader.ReadToEnd();
+                Console.WriteLine(responseContent);
+
                 XmlDocument doc = new XmlDocument();
```

Comment:
I would rename this to `responseXmlDoc` or something like that, rather than the very unspecific `doc`.